Notes and Outlines/markdown modules/02-editing.md

```diff
@@ -0,0 +1,77 @@
+# Editing Your GitHub Page
+
+We've set you up with a project to get started with. Now let's see how to make some edits to it. If you're not clear on the GitHub workflow, we recommend that you complete the [Wheelhouse GitHub Workflow course](https://learn.wheelhouse.io/events/workflow) first. 
+
+## Profile vs. Project Pages
+
+There are two kinds of GitHub Page projects: profile pages, and project pages. Every personal and organizational GitHub account gets one profile page. Profile pages are displayed from the `master` branch, and so are managed just like any other GitHub project. Your profile page must have the title of `username.github.io`, where `username` is your GitHub username or the name of the organization. To see your profile page, just go to the same URL as the title, so `http://username.github.io`.
+
```

Comment:
:+1:  Yeah, that makes sense to keep it consistent across GitHub.  It can get pretty confusing to describe how different accounts work with different types of GitHub Pages sites. I like the concept of "profile pages" though! It just gets murky when it comes to other aspects of GitHub.